### ISSUE TITLE
create method for easily changing data and testing in /local

### ISSUE THREAD

**Issue by sujato** [suttacentral/suttacentral#2795]
Currently if we want to test the effect of changing data, we must:

- make the data change in `sc-data`
- push to Github
- `make run-dev`
- `make load-data`

The latter command takes a long time to run! 

Is there some way we could simply make local changes to data and test them in `/local` without pushing to Github?

**Comment by ihongda** [comment]
@sujato  Bhante, you can try the steps:
1. Modify data in the /server/sc-data directory.
2. Execute `make load-data-no-pull`.

**Comment by thesunshade** [comment]
I wonder if this has been solved? @buddhist-uni?

**Comment by khemarato** [comment]
Yes and no. `load-data-no-pull` still takes a while?  But is this really an action that we do so often that it's worth optimizing?

**Comment by thesunshade** [comment]
> The latter command takes a long time to run!

For me this takes around two minutes. Is that a long time?

**Comment by mtnadi** [comment]
Possibly not the most useful info, but for me `make load-data-no-pull` took just over a minute, after I hard-coded-out loading HTML texts because it's somehow broken on my machine, but also seeing as I wasn't working with them.

(I don't know much how to work with docker, and haven't figured out how to usefully get into it to edit in `/server/sc-data`.  In my working directory, `/server/sc-data` remains empty.)
